utils/anaInfo.py

```diff
@@ -12,6 +12,25 @@
 
 import string
 
+#: Nominal current and voltage values from Tables 9 and 10 of the VFAT3 manual
+nominalDacValues = {
+        "CFG_IREF":(10,"uA"),
+        "CFG_BIAS_PRE_I_BIT":(150,"uA"),
+        "CFG_BIAS_PRE_I_BLCC":(25,"nA"),
+        "CFG_BIAS_SH_I_BFCAS":(26,"uA"),
+        "CFG_BIAS_SH_I_BDIFF":(16,"uA"),
+        "CFG_BIAS_SD_I_BDIFF":(28,"uA"),
+        "CFG_BIAS_SD_I_BFCAS":(27,"uA"),
+        "CFG_BIAS_SD_I_BSF":(30,"uA"),
+        "CFG_BIAS_CFD_DAC_1":(20,"uA"),
+        "CFG_BIAS_CFD_DAC_2":(20,"uA"),
+        "CFG_HYST":(100,"nA"),
+        "CFG_THR_ARM_DAC":(3.2,"uA"),
+        "CFG_THR_ZCC_DAC":(275,"nA"),
+        "CFG_BIAS_PRE_VREF":(430,'mV'),
+        "CFG_ADC_VREF":(400,'mV')
```

Comment:
I think we've tried to guess at a few of these values and have paired them improperly.  For example `CFG_ADC_VREF` should be as close to 1.0V as possible, see VFAT3 manual, section 9.1.4.

Additionally these values appear to match for VFAT3b, in the future (not now) we should think how to handle VFAT3a.  Certainly this week we will need to...but hopefully we can get some input from the VFAT3 designers.

Reply:
sorry, it was not clear to me if you are requesting any change right now

Reply:
Yes I guessed as much, but please review the table I sent for accuracy.  For example you've paired `CFG_ADC_VREF`, nominal value 1.0V as mentioned, incorrectly to the nominal value of `SLVS_Vref` e.g. 400mV.  This should be resolved and a double check of the rest of the table just to make sure.